frontend/src/components/dialogs/AssetDetailDialog.vue

```diff
@@ -45,6 +49,13 @@ export default {
         flow () {
             return this.payload?.flows?.flows || []
         },
+        nrVersion () {
+            const mods = this.payload?.settings?.modules
+            if (mods) {
+                return mods['node-red'] || 'ben'
```

Comment:
Assuming this was a placeholder that wasn't updated, "Unavailable" a better backup?

Reply:
yes, pushed update